cdk/setup.py

```diff
@@ -40,7 +40,7 @@
         f"aws-cdk.aws-sns=={CDK_VERSION}",
         f"aws-cdk.aws-ssm=={CDK_VERSION}",
         f"aws-cdk.core=={CDK_VERSION}",
-        f"oe-patterns-cdk-common@git+https://github.com/ordinaryexperts/aws-marketplace-oe-patterns-cdk-common@1.0.0"
+        f"oe-patterns-cdk-common@git+https://github.com/ordinaryexperts/aws-marketplace-oe-patterns-cdk-common@feature/cdk-upgrade-1.57.0"
```

Comment:
Looks good other than we should release a new version of the CDK Common (I just approved that PR) and then upgrade to that version here.